app/lib/menu.rb

```diff
@@ -0,0 +1,17 @@
+require 'json'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.